spec/requests/users/manager_spec.rb

```diff
@@ -45,13 +45,20 @@
 
     let(:new_user) { User.find_by(email: 'created@example.com') }
     let(:request) { post '/users', params: user_params }
-    # let(:outbox) { ActionMailer::Base.deliveries }
 
     it 'redirects to user_path' do
       request
       expect(response).to redirect_to user_path(new_user)
     end
 
+    it 'sends password reset email' do
+      message_delivery = instance_double(GovukNotifyRails::Mailer::MessageDelivery)
+      allow(Devise::Mailer).to receive(:reset_password_instructions).and_return(message_delivery)
+      allow(message_delivery).to receive(:deliver_later)
```

Comment:
to prevent clutter in the example it might be better to have this in a `before` block, unless this creates uneccessary overhead (could have a context block with a nested `before` block.

Reply:
👍